tests/testthat/test-plot_step_ahead_model_output.R

```diff
@@ -300,6 +303,103 @@ test_that("Output", {
 
 })
 
+test_that("sample output type functionality", {
```

Comment:
General comment on the tests. There is a large number of tests testing a lot of functionality but they are split into only 3 blocks. Would be much easier to follow if there were more more blocks with explicit description of what they are testing